localization/i18n/pl/OrcaSlicer_pl.po

```diff
@@ -18068,17 +18132,17 @@ msgid ""
 "Warning: The brim type is not set to \"painted\",the brim ears will not take "
 "effect !"
 msgstr ""
-"Ostrzeżenie: rodzaj obrzeża nie jest ustawiony na „malowane”. Uszy obrzeża "
-"nie będą działać!"
+"Ostrzeżenie: rodzaj brimu nie jest ustawiony na „malowane”. Uszy brim nie "
+"będą działać!"
 
 msgid "Set the brim type to \"painted\""
-msgstr "Ustal rodzaj obrzeża na „malowane“"
+msgstr "Ustaw rodzaj brimu na „malowane“"
 
 msgid " invalid brim ears"
-msgstr "nieprawidłowe uszy obrzeża"
+msgstr "nieprawidłowe uszy brim"
```

Comment:
dunno if this is important but here in original msgid there is a space at start.

Reply:
With great software comes great translations ;)